static/js/components/ClassificationForm.jsx

```diff
@@ -135,73 +133,69 @@ const ClassificationForm = ({ obj_id, taxonomyList }) => {
   // Custom form widget for probability because rxjs MUI UpdownWidget does not have working min/max/step
   // https://github.com/rjsf-team/react-jsonschema-form/issues/2022
   const CustomProbabilityWidget = ({ value, onChange }) => {
-    return (
-      <TextField
-        id="probability"
-        label="Probability"
-        type="number"
-        helperText="[0-1]"
-        InputLabelProps={{
-          shrink: true,
-        }}
-        inputProps={{ min: "0", max: "1", step: "0.0001" }}
-        value={value || ""}
-        onChange={(event) => {
-          onChange(event.target.value);
-        }}
-      />
-    );
+    <TextField
+      id="probability"
+      label="Probability"
+      type="number"
+      helperText="[0-1]"
+      InputLabelProps={{
+        shrink: true,
+      }}
+      inputProps={{ min: "0", max: "1", step: "0.0001" }}
+      value={value || ""}
+      onChange={(event) => {
+        onChange(event.target.value);
+      }}
+    />;
   };
   CustomProbabilityWidget.propTypes = {
     value: PropTypes.string.isRequired,
     onChange: PropTypes.func.isRequired,
   };
 
   const CustomGroupsWidget = ({ value, onChange, options }) => {
-    return (
-      <>
-        <InputLabel id="classificationGroupSelectLabel">
-          Choose Group (all groups if blank)
-        </InputLabel>
-        <Select
-          id="groupSelect"
-          onChange={(event) => {
-            onChange(event.target.value);
-          }}
-          input={<Input id="selectGroupsChip" />}
-          labelId="classificationGroupSelectLabel"
-          value={value || ""}
-          renderValue={(selected) => (
-            <div className={classes.chips}>
-              {selected.map((group) => (
-                <Chip
-                  key={group}
-                  label={groupIDToName[group]}
-                  className={classes.chip}
-                />
-              ))}
-            </div>
-          )}
-          MenuProps={MenuProps}
-          fullWidth
-          multiple
-        >
-          {options.enumOptions.length > 0 &&
-            options.enumOptions.map((group) => (
-              <MenuItem
-                value={group.value}
-                key={group.value.toString()}
-                data-testid={`notificationGroupSelect_${group.value}`}
-              >
-                {group.label}
-              </MenuItem>
+    <>
```

Comment:
@guynir42 here's the critical bug -- there's no return statement here

Reply:
Good catch! 

I removed the return statement because of the new lint rules about arrow functions with a single statement. But I needed to also remove the braces and semicolon. 
Now I'm back to where a few unrelated tests are still flakey...